Zend/zend_execute.c

```diff
@@ -3841,6 +3841,138 @@ static zend_always_inline zend_generator *zend_get_running_generator(EXECUTE_DAT
 }
 /* }}} */
 
+ZEND_API void zend_unfinished_calls_gc(zend_execute_data *execute_data, zend_execute_data *call, uint32_t op_num, zend_get_gc_buffer *buf) /* {{{ */
```

Comment:
This duplicates cleanup_unfinished_calls(). I will look into ways to factor this out in an other PR.

Reply:
I will try to reuse this in the fix for https://github.com/php/php-src/issues/9735 before I merge this.

Reply:
Is it possible to simplify cleanup_unfinished_calls() in a way that it calls this function with a buffer, then frees all zvals on that buffer?

Reply:
I will try this